src/tests.ts

```diff
@@ -1954,3 +2054,20 @@ type Test = {
 	expect(n: number): void
 	done(): void
 }
+
+type EasingFunctionGroup = {
+	In(amount: number): number
+	Out(amount: number): number
+	InOut(amount: number): number
+}
```

Comment:
Cool. I suppose this can be moved to the definition of the function groups in https://github.com/tweenjs/tween.js/blob/master/src/Easing.ts, then imported here.

Reply:
During the writing of this test, I considered how to add the definition to `Easing.ts`.

In order to add the EasingFunctionGroup type definition, we had to modify the export of index.ts.
I thought it would be excessive to modify index.ts to add tests, so I added a type definition to test.ts.
In the future, I also think that adding a type definition to Easing.ts is the best way to go.